src/plugins/workspace/public/components/home_get_start_card/use_case_footer.tsx

```diff
@@ -94,7 +94,7 @@ export const UseCaseFooter = ({
           onClick={showModal}
           data-test-subj="useCase.footer.createWorkspace.button"
         >
-          <FormattedMessage id="useCase.footer.createWorkspace" defaultMessage="Create workspace" />
+          {i18n.translate('useCase.footer.createWorkspace', { defaultMessage: 'Create workspace' })}
```

Comment:
```suggestion
          {i18n.translate('workspace.useCase.footer.createWorkspace', { defaultMessage: 'Create workspace' })}
```
The key should comply with the directory the file is in.

Reply:
thx!